tests/precalculations/test_lda_embedding.py

```diff
@@ -0,0 +1,198 @@
+import unittest
+from shift_detector.precalculations.lda_embedding import LdaEmbedding
+from shift_detector.precalculations.store import Store
+import pandas as pd
+
+
+class TestLdaEmbedding(unittest.TestCase):
+
+    def setUp(self):
+        self.lda1 = LdaEmbedding(cols=['text'], n_topics=2, n_iter=1, random_state=2, lib='sklearn')
+        self.lda2 = LdaEmbedding(cols=['text'], n_topics=2, n_iter=1, random_state=2, lib='sklearn')
+        self.lda3 = LdaEmbedding(cols=['text'], n_topics=2, n_iter=1, random_state=2, lib='gensim')
+        self.lda4 = LdaEmbedding(cols=['text'], n_topics=2, n_iter=1, random_state=2, lib='lda')
+        self.lda5 = LdaEmbedding(cols=['text', 'abc'], n_topics=2, n_iter=1, random_state=2, lib='sklearn')
+
+        self.poems = [
```

Comment:
Is it possible to use the data library?

Reply:
will be done in later pull request to be consistent with all tests